### ISSUE TITLE
Lost model WavLM

### ISSUE THREAD

**Issue by huihuihui-wq** [ShmuelRonen/ComfyUI-FreeVC_wrapper#4]
got prompt
FreeVC Node initialized on device: cuda
Starting voice conversion...
Processing with model: FreeVC
Loading FreeVC...
Loading config from: /data/home/liuruofei/ComfyUI/custom_nodes/ComfyUI-FreeVC_wrapper/freevc/configs/freevc.json
Initializing synthesis model...
/data/home/liuruofei/miniconda3/lib/python3.12/site-packages/torch/nn/utils/weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
[]
Loading checkpoint from: /data/home/liuruofei/ComfyUI/custom_nodes/ComfyUI-FreeVC_wrapper/freevc/checkpoints/freevc.pth
Loading speaker encoder...
Loaded the voice encoder model on cuda in 0.02 seconds.
Loading WavLM...
Downloading WavLM model if needed...

**Comment by ShmuelRonen** [comment]
Pre download the models as  describe in the readme file

**Comment by huihuihui-wq** [comment]
What folder should WavLM be placed in

**Comment by ShmuelRonen** [comment]
## Here:
![Image](https://github.com/user-attachments/assets/15fc6b18-ba42-4e80-8fbb-59e6fc1012f4)

## And here:
![Image](https://github.com/user-attachments/assets/6bb37442-08a4-49a3-a3fc-94a35e8faafb)
